### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by Njoe00**
Creating footer, text columns, social media icons, and bottom footer icons

### CODE REVIEW

File: src/components/SocialMediaFooterIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+export default function SocialMediaFooterIcon({
+  icon,
+}: {
+  icon: IconDefinition;
+}) {
+  const [changeColourOnHover, SetchangeColourOnHover] = useState(false);
```
Comment (MKCMMSK) [inline src/components/SocialMediaFooterIcon.tsx]: this isn't very clear, I would use something like isHovered

File: src/components/SiteMapFooter.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import FooterGotQuestionsBanner from "@/components/FooterGotQuestionsBanner";
+
+export default function SiteMapFooter() {
+  return (
+    <div className="bg-background-site-footer-colour">
+      <div className="px-14 pb-5 gap-2 pt-6 font-sans w-full h-[430px] flex flex-row justify-center text-gray-400 font-[400]">
+        {footerContent.map((content, index) => (
+          <ul className="flex-column text-xs w-56 h-6" key={index}>
+            {content.title}
+            <SiteMapFooterTextContent content={content.body} />
```
Comment (MKCMMSK) [inline src/components/SiteMapFooter.tsx:13]: I would rename this a little, it comes off as text but maybe it's bodyContent

**Review by MKCMMSK — CHANGES_REQUESTED** [review]

**Comment by MKCMMSK** [comment]
Why are the icons added to the PR? They aren't used anywhere. Are you using ```git add .``` ?

File: src/components/SocialMediaFooterIcon.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/SocialMediaFooterIcon.tsx]: addressed.

File: src/components/SiteMapFooter.tsx (hunk 2 above)
Comment (Njoe00) [reply src/components/SiteMapFooter.tsx:13]: addressed.

**Comment by Njoe00** [comment]
> Why are the icons added to the PR? They aren't used anywhere. Are you using `git add .` ?

Correct.

File: src/components/SocialMediaFooterIcon.tsx (hunk 1 above)
Comment (MKCMMSK) [reply src/components/SocialMediaFooterIcon.tsx]: from the other PR, I mentioned using is____ naming convention

**Review by MKCMMSK — CHANGES_REQUESTED** [review]

File: src/components/SocialMediaFooterIcon.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/SocialMediaFooterIcon.tsx]: addressed correctly.

**Pull request merged**